tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot

```diff
@@ -6,20 +6,65 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Default Tags
 
 *** Test Cases ***
-Display HTML
+Get Login Page
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/authentication
+    Should contain  ${output}  <title>VCH Admin</title>
+
+While Logged Out Fail To Display HTML
     ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}
+    Should not contain  ${output}  <title>VIC: ${vch-name}</title>
+
+While Logged Out Fail To Get Portlayer Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/port-layer.log
+    Should Not Contain  ${output}  Launching portlayer server
+
+While Logged Out Fail To Get VCH-Init Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/init.log
+    Should not contain  ${output}  reaping child processes
+
+While Logged Out Fail To Get Docker Personality Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/docker-personality.log
+    Should not contain  ${output}  docker personality
+
+While Logged Out Fail To Get Container Logs
+    ${rc}  ${output}=  Run And Return Rc and Output  docker ${params} pull busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${container}=  Run And Return Rc and Output  docker ${params} create busybox /bin/top
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${container}  Error
+    ${rc}  ${output}=  Run And Return Rc and Output  docker ${params} start ${container}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk ${vic-admin}/container-logs.tar.gz | tar tvzf -
+    Should not Be Equal As Integers  ${rc}  0
+    Log  ${output}
+    Should not Contain  ${output}  ${container}/vmware.log
+    Should not Contain  ${output}  ${container}/tether.debug
+
+While Logged Out Fail To Get VICAdmin Log
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/logs/vicadmin.log
+    Log  ${output}
+    Should not contain  ${output}  Launching vicadmin pprof server
+
+Login
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -sk ${vic-admin}/authentication -XPOST -F username=%{GOVC_USERNAME} -F password=%{GOVC_PASSWORD} -D /tmp/cookies-${vch-name}
+    Should Be Equal As Integers  ${rc}  0
```

Comment:
Leave this the way it is for now, but please create a follow on issue and assign it to me for me to clean this up and not have tests rely on previous tests.